deployment/aws-batch/job-definitions/pfb-analysis-run-job.json

```diff
@@ -4,15 +4,36 @@
     "parameters": {},
     "containerProperties": {
         "image": "{image}",
-        "vcpus": 4,
-        "memory": 4096,
+        "vcpus": 2,
+        "memory": 12288,
         "command": [
             "/pfb/scripts/entrypoint.sh"
         ],
         "jobRoleArn": "",
-        "volumes": [],
-        "environment": [],
-        "mountPoints": [],
+        "volumes": [{
+            "name": "pgdata",
+            "host": {
+                "sourcePath": "/media/nvme0n1/pgdata"
```

Comment:
Did this path get created automatically when you booted the `i3` (RE: `nvme0n1`)?